docs/changelog.rst

```diff
@@ -6,6 +6,11 @@ Changelog
 `Unreleased`_
 -------------
 
+Added
+~~~~~
+
+- support for property-level examples. `#637`_
```

Comment:
Should it be 467 as added below?

Reply:
Ah, I couldn't remember if it was issue number or PR number.